packages/integration-test/src/defaultCredsInvalidDisplayCheck.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { Application } from "spectron";
+import { awsCredPath, setupElectronLogs, startApp, stopApp } from "./hooks";
+import { TestRunner } from "./testRunner";
+import fs from "fs";
+
+describe("default creds check, check AWS access key displayed", () => {
+    let app: Application;
+    let runner: TestRunner;
+
+    beforeAll(async () => {
+        process.env["AWS_ACCESS_KEY_ID"] = "InValid";
+        process.env["AWS_SECRET_ACCESS_KEY"] = "InValid";
+        app = await startApp();
+        runner = new TestRunner(app);
+        return app;
+    })
+
+    beforeEach(async () => {
+        await app.client.refresh();
+    })
+
+    afterEach(async () => {
+        setupElectronLogs(app);
+        await app.client.pause(1000);
+        await app.client.refresh();
+    });
+
+    afterAll(async () => {
+        await app.client.execute("window.test.clearState();");
+        await stopApp(app);
+    });
+
+    test("set default credentials with invalid key", async () => {
+        await app.client.pause(3000);
+        await (await app.client.$("#start-btn")).click();
+        await runner.selectDefaultCredentials();
+        await runner.validateComponentExists('div=InValid');
```

Comment:
It might be good to have this in a separate test, but you could also validate the access key is showing in your other test before you hit next. Just a comment, not really a suggestion haha.

Reply:
This test does exactly that, sets the AWS Profile as Invalid and then check whether it's displayed. Do you mean something else?

Reply:
I mean that you have two separate integration tests, one seems like it checks for the access key to display, and the other checks the validation portion. It seems like you could just put them together, but I could see a reason to try to keep them separate to simplify each test.